### ISSUE TITLE
Finetuning

### ISSUE THREAD

**Issue by Rohan138** [mukobi/Unadaptable-Foundation-Models#6]
Implement finetuning for LLMs after #1 and #2; since we're doing basic SFT finetuning on known datasets, we can probaly reuse tooling like [unsloth](https://github.com/unslothai/unsloth) for full-parameter finetuning on e.g. test set of WMDP.

**Comment by Rohan138** [comment]
Duplicate of #21
